server/services/overviewPageService.ts

```diff
@@ -46,6 +66,67 @@ export default class OverviewPageService {
     }
   }
 
+  public async getStaffContacts(prisonerData: Prisoner): Promise<StaffContacts> {
+    const { bookingId } = prisonerData
+    const [offenderContacts, allocationManager, offenderKeyWorker, keyWorkerSessions] = await Promise.all([
+      this.prisonApiClient.getOffenderContacts(prisonerData.bookingId),
+      this.allocationManagerClient.getPomByOffenderNo(prisonerData.prisonerNumber).catch(() => undefined),
```

Comment:
Is there a reason this one has the `.catch()` where the others do not? it causes that bit below (lines 88-92) where you have to contend with it being `undefined` and also of `any` type so I wondered if it was necessary to catch it here where we don't any of the others.